### PR TITLE
chore: publish version 0.1.6

### PR DESCRIPTION
**Pull request by DanMossa**
Publish new version of gotrue-dart to be used inside of supabase-dart

### CODE REVIEW

**Comment by DanMossa** [comment]
@bdlukaa Does this look alright?

**Review by derakula — COMMENTED** [review]
up up asap with supabase flutter.. we need native google signin

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ## [next]
```
Comment (dshukertjr) [inline CHANGELOG.md:2]: We can get rid of this `[next]` piece. This is just a placeholder for when we publish new version.

**Review by dshukertjr — CHANGES_REQUESTED** [review]
Just one small change within the changelog, and  it looks good to me!

**Comment by DanMossa** [comment]
@dshukertjr Done!

**Review by dshukertjr — APPROVED** [review]
LGTM!

**Pull request merged**